src/types.ts

```diff
@@ -1,15 +1,15 @@
 import _ from 'lodash';
 import Long from 'long';
-import {Ydb} from "../proto/bundle";
+import {google, Ydb} from "../proto/bundle";
 import 'reflect-metadata';
 
-import Type = Ydb.Type;
-import IType = Ydb.IType;
-import IStructMember = Ydb.IStructMember;
-import IValue = Ydb.IValue;
-import IColumn = Ydb.IColumn;
-import ITypedValue = Ydb.ITypedValue;
-import IResultSet = Ydb.IResultSet;
+const Type = Ydb.Type;
```

Comment:
I had to change the `import` keyword to `const` and `type` to make jest test work.